readme.txt

```diff
@@ -37,6 +37,9 @@ Not in the way you're probably thinking. The markup surrounding the widget area
 
 == Changelog ==
 
+= 2.2.1 =
+* Allow sidebars to show in the custom Blog Posts page.
```

Comment:
```suggestion
* Allow sidebars to show in the WordPress Posts page.
```

Updated to reflect that this is a WordPress setting; used case used on `wp-admin/options-reading.php`